{"name": "Переход в раздел «История заказов» для залогиненного пользователя", "status": "broken", "statusDetails": {"message": "selenium.common.exceptions.TimeoutException: Message:", "trace": "driver = <selenium.webdriver.firefox.webdriver.WebDriver (session=\"0416da7a-191c-4522-b939-40bfd2ac1cc8\")>\ngenerate_random_user = {'email': 'ijohnson@example.com', 'name': 'Michelle Williams', 'password': '2434215136'}\n\n    @allure.step(\"Логин пользователя на сайте\")\n    @pytest.fixture()\n    def get_authorized(driver, generate_random_user):\n        HeaderPage(driver).click_my_account()\n>       LoginPage(driver).login_user(generate_random_user)\n\nconftest.py:47: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _\npages\\login_page.py:38: in login_user\n    self.wait_for_url_change(old_url)\npages\\base_page.py:58: in wait_for_url_change\n    return WebDriverWait(self.driver, 15).until(expected_conditions.url_changes(old_url))\n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _\n\nself = <selenium.webdriver.support.wait.WebDriverWait (session=\"0416da7a-191c-4522-b939-40bfd2ac1cc8\")>, method = <function url_changes.<locals>._predicate at 0x000001D9C67AAA70>\nmessage = ''\n\n    def until(self, method: Callable[[D], Union[Literal[False], T]], message: str = \"\") -> T:\n        \"\"\"Calls the method provided with the driver as an argument until the \\\n        return value does not evaluate to ``False``.\n    \n        :param method: callable(WebDriver)\n        :param message: optional message for :exc:`TimeoutException`\n        :returns: the result of the last call to `method`\n        :raises: :exc:`selenium.common.exceptions.TimeoutException` if timeout occurs\n        \"\"\"\n        screen = None\n        stacktrace = None\n    \n        end_time = time.monotonic() + self._timeout\n        while True:\n            try:\n                value = method(self._driver)\n                if value:\n                    return value\n            except self._ignored_exceptions as exc:\n                screen = getattr(exc, \"screen\", None)\n                stacktrace = getattr(exc, \"stacktrace\", None)\n            time.sleep(self._poll)\n            if time.monotonic() > end_time:\n                break\n>       raise TimeoutException(message, screen, stacktrace)\nE       selenium.common.exceptions.TimeoutException: Message:\n\n.venv\\lib\\site-packages\\selenium\\webdriver\\support\\wait.py:105: TimeoutException"}, "description": "Открывается страница истории заказов; есть нужный URL", "parameters": [{"name": "driver", "value": "'firefox'"}], "start": 1719831806826, "stop": 1719831806826, "uuid": "395c3ecd-d548-479c-bda2-a586c5382e26", "historyId": "279a50bacdc80f5337dcfec1b0c4500c", "testCaseId": "3283d97afa51a1d9da3587d14dd0f3bc", "fullName": "tests.test_my_account.TestMyAccount#test_transition_to_history_if_logged_in", "labels": [{"name": "parentSuite", "value": "tests"}, {"name": "suite", "value": "test_my_account"}, {"name": "subSuite", "value": "TestMyAccount"}, {"name": "host", "value": "XPMEPODW0067"}, {"name": "thread", "value": "19988-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.test_my_account"}]}